crates/sallyport/src/guest/call/enarxcall/types/result.rs

```diff
@@ -43,3 +43,13 @@ impl From<Result<usize>> for crate::Result<usize> {
         }
     }
 }
+
+impl From<Result<c_int>> for crate::Result<c_int> {
+    #[inline]
+    fn from(res: Result<c_int>) -> Self {
+        match res.0 {
+            errno @ ERRNO_START..=usize::MAX => Err(-(errno as c_int)),
```

Comment:
`errno @ ERRNO_START.. =>` should suffice.

Reply:
copy & pasted from existing sallyport code, but you are right.

Reply:
I believe unbounded ranges were unstable when most of sallyport was written.
We should add a commit/PR addressing all occurences

Reply:
pushed a fixup